src/site/layouts/vet_center_locations_list.drupal.liquid

```diff
@@ -14,9 +17,60 @@
 
           {% if fieldIntroText != empty %}
             <div class="va-introtext">
-              <p>{{ fieldIntroText }}</p>
+              <p class="vads-u-font-size--lg">{{ fieldIntroText }}</p>
             </div>
           {% endif %}
+
+            <h2 class="vads-u-font-size--xl vads-u-margin-top--3 medium-screen:vads-u-margin-top--5 vads-u-margin-bottom--2p5
+          medium-screen:vads-u-margin-bottom--3" id="main-location">
+                Main location
+            </h2>
+            {% include "src/site/includes/vet_center_address_phone_image.liquid" with
+                vetCenter = fieldOffice.entity
+                vetCenterUrl = entityUrl.breadcrumb.1.url.path %}
+
+            {% if fieldOffice.entity.reverseFieldOfficeNode.entities.length > 0 %}
+            <h2 class="vads-u-font-size--xl vads-u-margin-top--3 medium-screen:vads-u-margin-top--5 vads-u-margin-bottom--2p5
+                      medium-screen:vads-u-margin-bottom--4" id="satellite-locations">
+                Satellite locations
+            </h2>
+            <p class="vads-u-font-size--base vads-u-margin-bottom--2p5
+                  medium-screen:vads-u-margin-bottom--4"> If you can’t make it to our {{ fieldOffice.entity.title }} we
+                offer satellite locations that may be closer to you.
+                These satellite facilities provide select services with the same community, care, and confidentiality in
+                a non-medical setting.
+                Call us for more information about these locations.
+            </p>
+            {% for entityVetCenter in fieldOffice.entity.reverseFieldOfficeNode.entities %}
+                {% include "src/site/includes/vet_center_address_phone_image.liquid" with
+                    vetCenter = entityVetCenter
+                    mainVetCenterPhone = fieldOffice.entity.fieldPhoneNumber %}
+            {% endfor %}
+            {% endfor %}
+
+            {% if fieldNearbyVetCenters.length > 0 %}
+            <h2 class="vads-u-font-size--xl vads-u-margin-top--3 medium-screen:vads-u-margin-top--5 vads-u-margin-bottom--2p5
+                 medium-screen:vads-u-margin-bottom--3" id="other-near-locations">
+                Other nearby Vet Centers
+            </h2>
```

Comment:
This `h2` should be indented. Have you tried using a plugin to auto-format the code?